scripts/desktop-launch.sh

```diff
@@ -1,8 +1,12 @@
-#!/bin/sh
+#!/bin/bash
```

Comment:
Is this change required? I tested the new lines with /bin/sh (which points to dash) and they work fine.